File: output/gemma4-e4b/avatar/Python/Java/atcoder_AGC002_A.java
import java.util.Scanner;

public class atcoder_AGC002_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read two integers a and b
        if (!scanner.hasNextInt()) {
            // Handle case where no input is provided, though typically competitive programming assumes valid input
            return;
        }
        
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        
        scanner.close();

        if (a > 0) {
            System.out.println("Positive");
        } else if (a <= 0 && b >= 0) {
            System.out.println("Zero");
        } else {
            // The original Python logic uses (a + b) % 2 == 0 for the 'Negative' case
            if ((a + b) % 2 == 0) {
                System.out.println("Negative");
            } else {
                System.out.println("Positive");
            }
        }
    }
}

File: output/gemma4-e4b/avatar/Java/Python/atcoder_AGC002_A.py
import sys

def solve():
    try:
        # Read all inputs from stdin
        data = sys.stdin.read().split()
        if not data:
            return

        a = int(data[0])
        b = int(data[1])

        if a > 0:
            print("Positive")
        elif b >= 0:
            print("Zero")
        elif (b - a + 1) % 2 == 0:
            print("Positive")
        else:
            print("Negative")

    except EOFError:
        pass
    except Exception:
        pass

solve()